### PR TITLE
Added Extra faq questions

### PR DESCRIPTION
**Pull request by spoorthyv**

### CODE REVIEW

File: src/faq/plugin.ts [hunk 1]
```diff
@@ -88,8 +88,11 @@ function activateFAQ(app: Application): void {
...
           <li class="faqanswer"> Central to the project is the Jupyter Notebook, a web-based platform that allows users to combine live code, equations, narrative text, visualizations, interactive dashboards and other media. Together these building blocks make science and data reproducible across over 40 programming languages and combine to form what we call a computational narrative. </li>
           <li class="faqquestion"> How is JupyterLab different from the Jupyter Notebook? </li>
-          <li class="faqanswer"> JupyterLab allows users to arrange multiple jupyter notebooks, text editors, terminals, output areas, etc. on a single page with multiple panels and tabs into one application. The codebase and UI of JupyterLab is based on a flexible plugin system that makes it easy to extend with new components.
-          </li>
+          <li class="faqanswer"> JupyterLab allows users to arrange multiple jupyter notebooks, text editors, terminals, output areas, etc. on a single page with multiple panels and tabs into one application. The codebase and UI of JupyterLab is based on a flexible plugin system that makes it easy to extend with new components.</li>
```
Comment (afshin) [inline src/faq/plugin.ts]: "Jupyter" should be capitalized, I believe.

File: src/faq/plugin.ts [hunk 2]
```diff
@@ -88,8 +88,11 @@ function activateFAQ(app: Application): void {
...
+          <li class="faqquestion"> How Stable is JupyterLab? </li>
+          <li class="faqanswer"> JupyterLab is currently in a Alpha release and not ready for public use. New features and bug fixes are being added very frequently. However, testing, development, and user feedback are greatly appreciated. </li>
+          <li class="faqquestion"> I’m confused with the interface. How do I navigate around JupyterLab? </li>
+          <li class="faqanswer"> Checkout the Jupyter Lab tour (Look in the help secton). </li>
```
Comment (afshin) [inline src/faq/plugin.ts]: Typo: `secton` => `section`.

**Comment by spoorthyv** [comment]
@afshin We are making a bunch of changes right now based on @Carreau 's comments in the issue. I'm not sure how to link a page within the application through inner html.

File: src/faq/plugin.ts [hunk 3]
```diff
@@ -86,19 +86,35 @@ function activateFAQ(app: Application): void {
...
+        <li class="faqquestion"> I have security concerns about JupyterLab.</li>
+        <li class="faqanswer"> If you have any inquiries, concerns, or thought you found a security vulnerability, please write to us <a class="faqa" href="mailto:atsecurity@jupyter.org">atsecurity@jupyter.org</a>. We will do our best to respond to you promptly.  </li>
+        <li class="faqquestion"> How can I contribute? </li>
+        <li class="faqanswer"> There are many ways to contribute to JupyterLab. Whether you are an experienced python programmer or a newcomer, any interested developers are welcome. You can learn about the JupyterLab codebase by watching [Brian Granger’s Keynote] and [talk] at scipy 2016, as well as our tutorial walkthrough and documentation. Also, feel free to ask questions on our <a class="faqa" target="_blank" href="https://github.com/jupyter/jupyterlab">github</a> or through any of our <a class="faqa" target="_blank" href="http://jupyter.org/community.html">community resources</a>. </li>
```
Comment (blink1073) [inline src/faq/plugin.ts]: Missing links, are we waiting for an upload tomorrow?

**Comment by spoorthyv** [comment]
Yeah. Not sure what the best way to approach this is. Should I just wait till tomorrow after the keynote to merge or edit the question and merge asap?

**Comment by blink1073** [comment]
Let's wait, it should be up by mid-afternoon

**Comment by ellisonbg** [comment]
I am going to merge and take a pass at the content myself.

**Pull request merged**